hnn_core/viz.py

```diff
@@ -1185,3 +1198,234 @@ def plot_laminar_csd(times, data, contact_labels, ax=None, colorbar=True,
     plt_show(show)
 
     return ax.get_figure()
+
+
+class NetworkPlot:
```

Comment:
```suggestion
class NetworkDataPlotter:
```

?

Reply:
`NetworkPlotter`? It might useful too for just visualizing the 3D morphology without any data too

Reply:
Yeah if data is None